src/drf_yasg/inspectors.py

```diff
@@ -114,7 +114,8 @@ def get_view_serializer(self):
         """
         if not hasattr(self.view, 'get_serializer'):
             return None
-        return self.view.get_serializer()
```

Comment:
I don't follow, though. In [the link you sent](https://github.com/encode/django-rest-framework/blob/a81e60ff390b9b6788b2cac24a01ee4dd2dcffd2/rest_framework/generics.py#L111), you can see that `get_serializer` already adds the serializer context before instantiating `serializer_class`.

Reply:
damn, you are right!